File: poker-client/src/app/api/models/toast-model.ts
/* tslint:disable */
import {ToastClassModel} from './toast-class-model';

/**
 * Contains the message to display and the duration of the toast.
 */
export interface ToastModel {

  /**
   * The message to display.
   */
  message?: string;
  options?: ToastClassModel;
}

File: poker-server/src/main/java/com/poker/poker/models/websocket/ToastModel.java
package com.poker.poker.models.websocket;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Schema(description = "Contains the message to display and the duration of the toast.")
public class ToastModel {

  @Schema(description = "The message to display.", example = "Login Successful!")
  private String message;

  @Schema(implementation = ToastClassModel.class)
  private ToastClassModel options;
}
